src/main/java/org/apache/maven/plugins/assembly/mojos/AbstractAssemblyMojo.java

```diff
@@ -430,6 +430,15 @@ public abstract class AbstractAssemblyMojo extends AbstractMojo implements Assem
     @Parameter
     private String overrideGroupName;
 
+    /**
+     * Override of mask which is applied to permissions of files/directories before they are put into assembly.
+     * If {@code null} then the mask is not explicitly configured and remains implementation-specific.
+     * If invalid value is specified - like negative value - then behaviour is implementation specific, i.e. depends
```

Comment:
an invalid value
such as a negative value
the behavior
That is, it depends on the underlying
of assembly --> the assembly

Reply:
Thanks. Fixed.